cli/script/command-parser.ts

```diff
@@ -227,6 +236,7 @@ var argv = yargs.usage(USAGE_PREFIX + " <command>")
 
                 addCommonConfiguration(yargs);
             })
+            .command("clear", "Clears away all the updates associated with a deployment", (yargs: yargs.Argv) => deploymentClear("clear", yargs))
```

Comment:
I'd vote for "Clears the release history associated with a deployment.", that way, we ensure the details about what it is doing is clear without needing to encode it into the command name (e.g. "clear-history").